content/en/dashboards/functions/_index.md

```diff
@@ -54,3 +51,4 @@ Here is an example of how to apply a timeshift function on your error logs to co
 
```

Comment:
Should we add a Further Reading section here with some links defined in the front matter too?